test/test_helper.rb

```diff
@@ -6,11 +6,11 @@
 require 'minitest/autorun'
 
 def fake_ffprobe_version_path
-  File.expand_path('../', __FILE__) + "/fake_ffprobe_version"
+  File.expand_path('../', __FILE__) + '/fake_ffprobe_version'
 end
 
 def fake_ffprobe_output_path
-  File.expand_path('../', __FILE__) + "/fake_ffprobe_output"
+  File.expand_path('../', __FILE__) + '/fake_ffprobe_output'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.